app/locale/en.coffee

```diff
@@ -990,8 +990,10 @@
     getting_started_3: "You'll see student's progress below as they sign up and join your class."
     additional_resources: "Additional Resources"
     additional_resources_1_pref: "Download/print our"
-    additional_resources_1_mid: "Course 1 Teacher's Guide"
-    additional_resources_1_suff: "explanations and solutions to each level."
+    additional_resources_1_mid: "Course 1"
+    additional_resources_1_mid2: " and "
```

Comment:
We should use `.spr` and `.spl` in **teacher-courses-view.jade**, instead of putting spaces in this string.  String translations and page layout should be kept as independent as possible.